cves/CVE-2011-0722.yml

```diff
@@ -90,18 +100,18 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: ec10d2d53999f6edf7d7b5ac88df263eccfb1fb0
-  note: ''
-- commit: d90aeeaf569e4a08c30b3d1d09c3cff3a86eb431
```

Comment:
Was this removal intentional? The two code blocks differ, and I believe this is the commit that introduced the actual fix!

Reply:
It was intentional. While this commit does introduce some additional low-resolution-related code, I believe that the fix had already been handled in the first listed fix commit, and this one helped to resolve a different issue pertaining specifically to low resolution decoding.